WalletWasabi/JsonConverters/BlockCypherDateTimeOffsetJsonConverter.cs

```diff
@@ -4,32 +4,26 @@
 
 namespace WalletWasabi.JsonConverters
 {
-	public class BlockCypherDateTimeOffsetJsonConverter : JsonConverter
+	public class BlockCypherDateTimeOffsetJsonConverter : JsonConverter<DateTimeOffset>
 	{
 		/// <inheritdoc />
-		public override bool CanConvert(Type objectType)
-		{
-			return objectType == typeof(DateTimeOffset);
-		}
-
-		/// <inheritdoc />
-		public override object? ReadJson(JsonReader reader, Type objectType, object existingValue, JsonSerializer serializer)
+		public override DateTimeOffset ReadJson(JsonReader reader, Type objectType, DateTimeOffset existingValue, bool hasExistingValue, JsonSerializer serializer)
```

Comment:
Shouldn't it be nullable?

Reply:
I don't know why but this one shouldn't.

Reply:
@lontivero Check this https://github.com/zkSNACKs/WalletWasabi/pull/6805/commits/24f2af1a354e5bb83557e9a9fe125736e9a50b7c